code/modules/plumbing/plumbers/grinder_chemical.dm

```diff
@@ -56,11 +54,10 @@
 
 	var/obj/item/I = AM
 	var/result
-	if(I.grind_results || I.juice_typepath)
+	if(I.grind_results)
+		result = I.grind(reagents, usr)
+	else
+		result = I.juice(reagents, usr)
```

Comment:
Does it behave nicely when it doesn’t have a juice_results?

Reply:
Yup. As long as it has reagents it can juiced. It  mixes juice results with the reagents for the final result

Reply:
Yeah I’m all good, working a lot on finishing my degree. Excuse the slowness

You still fighting machine code I see?

Reply:
Yup all good here friend :D. Degree you say? I actually did my master's in computer science 2 years ago.

Still unemployed though but hey I'm having fun here so no regrets.

> You still having fun?

In the future I'm going to ping you to have plesent conversations like this so yes I am having fun. Thanks for asking.

See you soon

Reply:
Oh shit, that’s sick! Glad you’re having fun

Sure! Just hit me up whenever, might be slow for a while though

Reply:
Definitely look forward to it. My PRs take like 3 weeks and more to be merged so I can deal with slow no worries.

Here's a pearl of wisdom that will age like wine 🍷.

2 years ago you dropped a comment offering me help with my very 1st PR

https://github.com/tgstation/tgstation/pull/71018#issuecomment-1301062910

A few weeks later you responded back with some relatable and nice words

https://github.com/tgstation/tgstation/pull/71410#issuecomment-1322159781

People like me we don't forget kind gestures like these no matter how many years pass by.

Ever since then I've valued this long distance internet friendship we've had.

The Pearl of wisdom here is "A few kind words can go a long way". Keep staying in touch your university friends over the coming years. I didn't do that and it is one of my biggest regrets.

Always a pleasure talking to you

Reply:
You know what I'm marking this conversation as a safe space.

If you ever want to talk about life and stuff feel free to ping me here. Im always here

Reply:
Haha good memory! Don’t worry about me though, I’m doing fine

Reply:
While we’re here, I appreciate you working on plumbing and just generally fixing up old code, we don’t get a lot of that so it matters all the more